src/main/java/io/gravitee/gateway/jupiter/api/connector/ConnectorFactory.java

```diff
@@ -37,9 +38,12 @@
     Set<ConnectorMode> supportedModes();
 
     /**
-     * Allow creating new connector from the given string configuration
+     * Allows creating new connector from the given string configuration.
```

Comment:
```suggestion
     * Allow creating new connector from the given string configuration.
```
Allow was right actually. It is imperative.